receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_metadata.go

```diff
@@ -35,22 +35,31 @@ const (
 
 // descriptors
 
-var fileSystemUsageDescriptor = func() dataold.MetricDescriptor {
-	descriptor := dataold.NewMetricDescriptor()
-	descriptor.InitEmpty()
-	descriptor.SetName("system.filesystem.usage")
-	descriptor.SetDescription("Filesystem bytes used.")
-	descriptor.SetUnit("bytes")
-	descriptor.SetType(dataold.MetricTypeInt64)
-	return descriptor
+var fileSystemUsageDescriptor = func() pdata.Metric {
+	metric := pdata.NewMetric()
+	metric.InitEmpty()
+	metric.SetName("system.filesystem.usage")
+	metric.SetDescription("Filesystem bytes used.")
+	metric.SetUnit("bytes")
+	metric.SetDataType(pdata.MetricDataTypeIntSum)
+	sum := metric.IntSum()
+	sum.InitEmpty()
+	sum.SetIsMonotonic(false)
+	sum.SetAggregationTemporality(pdata.AggregationTemporalityCumulative)
+	return metric
 }()
 
-var fileSystemINodesUsageDescriptor = func() dataold.MetricDescriptor {
-	descriptor := dataold.NewMetricDescriptor()
-	descriptor.InitEmpty()
-	descriptor.SetName("system.filesystem.inodes.usage")
-	descriptor.SetDescription("FileSystem operations count.")
-	descriptor.SetUnit("1")
-	descriptor.SetType(dataold.MetricTypeInt64)
-	return descriptor
+var fileSystemINodesUsageDescriptor = func() pdata.Metric {
+	metric := pdata.NewMetric()
+	metric.InitEmpty()
+	metric.SetName("system.filesystem.inodes.usage")
+	metric.SetDescription("FileSystem operations count.")
+	metric.SetUnit("1")
+	metric.SetDataType(pdata.MetricDataTypeIntSum)
+	sum := metric.IntSum()
+	sum.InitEmpty()
+	// TODO: Investigate why not cumulative?
```

Comment:
I believe inodes can be freed?

Reply:
Are you asking... Don't know that's what I left the Todo, based on the description this is the number of ops so it is cumulative. Maybe you can check :)

Reply:
Yes that's what I was answering.

And oh yea the description is wrong 😮  Seems like that was copied from the disk operations metric by mistake. It should be "iNodes used."

Reply:
Done.